### DISCUSSION TITLE
Redirect to IDP based on login_hint/email query param

### DISCUSSION THREAD

**Discussion by dheeraj-armada** [sventorben/keycloak-home-idp-discovery#388] [Q&A] [answered]
I am not sure if this is a feature request or a question, in case if this already exists. I would like to redirect to the matching IDP not after the user has hit Login/Submit but based on email in the query parameter being passed while requesting the KC login page and the domain on that email.

Is this possible?

**Comment by sventorben** [comment, marked as answer]
Hey @dheeraj-armada 

please checke the docs, especially "Bypass login page" configuration option, at https://sventorben.github.io/keycloak-home-idp-discovery/configuration.html#general-configuration-options

Best,
Sven-Torben

**Reply by dheeraj-armada** [reply]
Hi @sventorben thanks for the reply. I had in fact went through the docs and the configuration options. But this was not working as expected in some scenarios which I am yet to identify exactly. Anyways, thanks for confirming.
